From 51f459b6a76b92d44873039804c3b0ba3bb23463 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Tue, 13 Oct 2015 16:00:57 +0800
Subject: [PATCH 0564/5242] MLK-11702-2 ARM: dts: imx6sx-sabreauto: add dts
 support for pxp v4l2 output

commit  6a86dd415e7ece7f17576034332db1d9c80904ff from
https://source.codeaurora.org/external/imx/linux-imx.git

Add dts support for pxp v4l2 output on imx6sx sabreauto
board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 1986e79..6c8f44e 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -37,6 +37,11 @@
 		reg = <0x80000000 0x80000000>;
 	};
 
+	pxp_v4l2_out {
+		compatible = "fsl,imx6sx-pxp-v4l2", "fsl,imx6sl-pxp-v4l2";
+		status = "okay";
+	};
+
 	leds {
 		compatible = "gpio-leds";
 		pinctrl-names = "default";
-- 
1.7.9.5

